whole-repo-tests/test_mypy.py

```diff
@@ -52,7 +52,7 @@ def get_mypy_analysed_type(fname, val):
     return (
         out.split("Revealed type is ")[1]
         .strip()
-        .strip('"' + "'")
+        .strip('"' + "'")  # noqa: ISC003
```

Comment:
Small thing, but `# noqa` comments like this can be confusing to later readers, requiring a search to work out what it does before you can tell whether it's a reasonable error to suppress.

In this case I can't think of a nice way to write equivalent code, so appending a `# string addition is OK here` comment would be ample clarification.